contracts/shared/AirdropBeta.sol

```diff
@@ -18,7 +18,7 @@ abstract contract AirdropBeta is AirdropAdmin {
         _;
     }
 
-    constructor(address _betaAddress) {
+    constructor(address _betaAddress, address _admin) AirdropAdmin(_admin) {
```

Comment:
I think that it is better to separate AirdropBeta and AirdropAdmin.
And in contract where both are used inherit from both
for example
`contract AirbroCampaignFactory is AirdropBeta, AirdropAdmin`